Clarify xk6-browser running instructions (#814)
grafana/k6-docs: src/data/markdown/docs/30 xk6-browser/30 xk6-browser.md

The quickest way to get started is to [download a release binary from GitHub](https://github.com/grafana/xk6-browser/releases).

We suggest this option if you don't want to build your own binary with [xk6](https://github.com/grafana/xk6), which can be challenging in some cases.

### Build from source

If you're more adventurous or want to get the latest changes of the xk6-browser extension, you can also build a binary from source.

> To build a binary with the extension, first, ensure you have installed [Go](https://golang.org/doc/install) and [Git](https://git-scm.com/). Then run the following commands in a terminal:

Reviewer: Putting instructions in ordered lists highlights that this is a procedure with a set sequence.

> To build a binary with the extension, first, ensure you have installed [Go](https://golang.org/doc/install) and [Git](https://git-scm.com/). Then run the following commands in a terminal:

Author: In general, I'd agree, but in this case I think it's better if we keep it consistent with the `InstallationInstructions` element.

Besides, it's a sequence of just 2 steps, and the first one is more of a pre-requisite we mention in passing. If we make it a standalone step, we might want to expand on how to confirm that Go and Git are installed properly, minimum versions, and things like that. Since this is rather off-topic for this document (and most of our installation instructions), I think it's fine if we leave this inline, as it was.

> To build a binary with the extension, first, ensure you have installed [Go](https://golang.org/doc/install) and [Git](https://git-scm.com/). Then run the following commands in a terminal:

Author: I disagree, not all instructions have to be formatted as a list, particularly not 1 or 2 step ones like here. I find the Go and Git list in the "Before you start" section on the "Output Extensions" page overly verbose, for no practical gain, IMO.

But this is a style preference, and you've been working on the broader docs more than me, so I'll defer to your opinion.

Updating the `InstallationInstructions` element is out of scope for this PR, but I'll accept your suggestion here.

If you have downloaded the pre-built binary you will find the binary named `xk6-browser`. Using the [example](#example), create a new file (such as `browser_test.js`) in the same directory as the pre-built binary, and paste the example in that file. Now run:

```
./xk6-browser run browser_test.js
```

> Note the `./` prefix, which tells your shell to run the binary located in the current working directory. This is required on macOS and Linux, but not on Windows if you're using the `cmd.exe` shell. If you're using PowerShell, you should specify `.\xk6-browser` instead.
>

> > If you installed xk6-browser via a system package, or you've placed the binary in a directory that's part of your `$PATH` environment variable, then you can omit the `./` or `.\` prefix.

Reviewer: Is this necessary information? My guess is that most people who want to browser test know this, especially if they know about $PATH variables.

But, I trust that the author knows the reader best.

I'm more open to leaving a note just for windows users, especially if `cmd.exe` and Powershell will fail with this syntax.

If we leave a note in, I suggest using the `<Blockquote>` Gatsby component to keep things consistent.

```
<Blockquote mod="note" title="A note for Windows users">

On Windows shells, the `./` file prefix will raise an exception.
Instead, try these:
- For `command.exe`, use no file prefix.
- In Powershell, use `.\xk6-browser`.

</Blockquote>
```

Will render as:

![image](https://user-images.githubusercontent.com/47385188/192040099-9f9fbe60-5cba-4171-b620-f7a42e002ba0.png)

Author: Resolved by 41c7e883.

If you have downloaded the pre-built binary you will find the binary named `xk6-browser`. Using the [example](#example), create a new file (such as `browser_test.js`) in the same directory as the pre-built binary, and paste the example in that file. Now run:

```
./xk6-browser run browser_test.js
```

<Blockquote mod="note" title="">

> Note the `./` prefix, which tells your shell to run the binary located in the current working directory. This is required on macOS and Linux, but not on Windows if you're using the `cmd.exe` shell. If you're using PowerShell, you should specify `.\xk6-browser` instead.
> 
> If you installed xk6-browser via a system package, or you've placed the binary in a directory that's part of your `$PATH` environment variable, then you can omit the `./` or `.\` prefix.

Reviewer: ```suggestion
The `./` prefix tells your shell to run the binary located in the current working directory. This is required on macOS and Linux, but not on the Windows `cmd.exe` shell. If you're using PowerShell, specify `.\xk6-browser` instead.

If you installed xk6-browser with a system package, or placed the binary in a directory that's part of your `$PATH` environment variable, you can omit the `./` or `.\` prefixes.
```

Reviewer: Small language trimming. I don't think we need to add the word "Note" anymore, since the new `blockquote` component  expresses that already, even for screenreaders.
